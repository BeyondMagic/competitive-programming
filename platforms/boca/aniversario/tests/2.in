2
Wilson 82
Natalia 28
