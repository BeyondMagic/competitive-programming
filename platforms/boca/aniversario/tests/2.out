Em 11 anos Natalia faz 39 e Wilson 93
Em 0 anos Wilson faz 82 e Natalia 28
